registrations/tests.py

```diff
@@ -3072,10 +3074,8 @@ def test_registration_process_clinic_good(self):
 
         # Check
         # . check number of calls made:
-        #   message set, schedule, popi message set, jembi registration,
-        #   id_store mother, id_store mother_reverse,
-        #   id_store registrant_rever, id_store patch
-        self.assertEqual(len(responses.calls), 8)
+        #   message set, schedule, popi message set, jembi registration
+        self.assertEqual(len(responses.calls), 4)
```

Comment:
Is this number ok? Looks like we're not doing the requests from `remove_personally_identifiable_fields` anymore?

Reply:
it was missing the request_jembi_url mock